scanspec/regions.py

```diff
@@ -1,29 +1,29 @@
 from dataclasses import dataclass, field, is_dataclass
-from typing import Any, Dict, Iterator, List, Set
+from typing import Any, Iterator, List, Set
 
 import numpy as np
-from apischema import schema
+from apischema import schema as api_schema
 
-from .core import Positions, Serializable, if_instance_do
+from .core import Points, Serializable, if_instance_do
 
 
 @dataclass
 class Region(Serializable):
     """Abstract baseclass for a Region that can `Mask` a `Spec`. Supports operators:
 
-    - ``|``: `UnionOf` two Regions, positions present in either
-    - ``&``: `IntersectionOf` two Regions, positions present in both
-    - ``-``: `DifferenceOf` two Regions, positions present in first not second
-    - ``^``: `SymmetricDifferenceOf` two Regions, positions present in one not both
+    - ``|``: `UnionOf` two Regions, points present in either
```

Comment:
Regions filter only on midpoints. This rename has shown we need to make this clear throughout...

Reply:
I did wonder about this one, hopefully I have updated it to be a bit clearer